src/components/UsernameInput/UsernameInput.tsx

```diff
@@ -28,9 +28,34 @@ const UsernameInput: FC<UsernameInputProps> = ({ state, onInputChange, label })
         console.log(color.toFormat("hex"));
     };
 
+    const handleAutoComplete = (variant: string) => {
+        switch (variant) {
+            case "name":
+                return { name: "name", type: "name", autoComplete: "name" };
+            case "title":
+                return { name: "organization-title", type: "organization-title", autoComplete: "organization-title" };
+            case "company":
+                return { name: "organization", type: "organization", autoComplete: "organization" };
+            case "email":
+                return { name: "email", type: "email", autoComplete: "email" };
+            case "phone":
+                return { name: "tel", type: "tel", autoComplete: "tel" };
+            case "linkedin":
+                return { name: "url", type: "url", autoComplete: "url" };
+            default:
+                return {};
+        }
+    };
```

Comment:
Okk, super, great start! 

Pero remember the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) - todos los `key: value` pairs en el return object tienen el mismo valor 

Podriamos hacer algo como 

```javascript 
let autocompleteVariant = "" ;

switch(variant) {

... // each case should have a `break;` at the end

}

return { 
  name: autocompleteVariant,
  type: autocompleteVariant,
  autoComplete: autocompleteVariant
}

```